connector/pom.xml

```diff
@@ -186,5 +186,24 @@
             <optional>true</optional>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-main</artifactId>
+            <version>405</version>
+            <scope>test</scope>
+        </dependency>
+        <!-- Maven Dependency -->
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-testing</artifactId>
+            <version>405</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
```

Comment:
We should not use AWS Java SDK 1.x.
AWS Java SDK 2 is already included in the dependency.